platform/android/java/app/config.gradle

```diff
@@ -127,7 +127,7 @@ final String VALUE_SEPARATOR_REGEX = "\\|"
 ext.getExportEnabledABIs = { ->
     String enabledABIs = project.hasProperty("export_enabled_abis") ? project.property("export_enabled_abis") : "";
     if (enabledABIs == null || enabledABIs.isEmpty()) {
-        enabledABIs = "armeabi-v7a|arm64-v8a|x86|x86_64|"
+        enabledABIs = "arm32|arm64|x86_32|x86_64|"
```

Comment:
This is not going to work as the Android build system expects specific terms for the ABIs it supports (https://developer.android.com/ndk/guides/abis#gc).